packages/jest-cli/src/runGlobalHook.js

```diff
@@ -6,9 +6,22 @@
  *
  * @flow
  */
+
 import type {GlobalConfig} from 'types/Config';
 import type {Test} from 'types/TestRunner';
 
+import {extname, resolve} from 'path';
+import pEachSeries from 'p-each-series';
+import {addHook} from 'pirates';
+import {ScriptTransformer} from 'jest-runtime';
+
+const inJestsSource = __dirname.includes('packages/jest-cli');
+let packagesRoot;
+
+if (inJestsSource) {
```

Comment:
this is really bad, but I'm not sure of another way to avoid trying to transpile babel-jest and our plugins themselves, which throws due to it being circular (needing babel-jest to transpile babel-jest)

Reply:
If I understand correctly, this problem only arises in our `global_{setup,teardown}` e2e tests.
What about adding `"transformIgnorePatterns": ["/node_modules/", "/packages/"]` to their `package.json` / `projects.jest.config.js` Jest configs?

Reply:
good call. It's not an issue since we don't use global setup/teardown in Jest, but I agree it's better to deal with it in this repo than putting it in the source code.